### ISSUE TITLE
list message showing sent but not received & same with button message 

### ISSUE THREAD

**Issue by pra021** [salman0ansari/whatsapp-api-nodejs#283]
list message showing sent but not received & same with button message, is any library updated?

**Comment by salman0ansari** [comment]
https://github.com/adiwajshing/Baileys/issues/2006

**Comment by sairmali** [comment]
+1

**Comment by sairmali** [comment]
How can we implement this ?

**Comment by sairmali** [comment]
@salman0ansari please give any solution ?

**Comment by salman0ansari** [comment]
> @salman0ansari please give any solution ?

Install Baileys edge version

**Comment by sairmali** [comment]
will i go in my project folder ?

cd whatsapp-api-nodejs/

then ->
yarn add @adiwajshing/baileys


can i follow this staple?

**Comment by sairmali** [comment]
please tell me @salman0ansari ?

**Comment by gbenmelej** [comment]
+1

**Comment by salman0ansari** [comment]
#190

**Comment by sairmali** [comment]
![image](https://user-images.githubusercontent.com/54973008/185729297-a991252a-3086-46e0-a44a-0058924b5167.png)

**Comment by sairmali** [comment]
![image](https://user-images.githubusercontent.com/54973008/185731466-d607d30a-69ef-48c9-86a8-ed1501f08d95.png)

**Comment by sairmali** [comment]
@salman0ansari this error i am facing

**Comment by salman0ansari** [comment]
use some brain
```
$ yarn add github:adiwajshing/baileys
$ cd node_modules/@adiwajshing/baileys
$ yarn
```

**Comment by salman0ansari** [comment]
fixed in latest commit 1b577d547bb46598aaf12e6fbfe434a2ff44c30d
